openmeetings-core/src/main/java/org/apache/openmeetings/core/remote/KStream.java

```diff
@@ -324,6 +319,16 @@ private WebRtcEndpoint createEndpoint(String sid, String uid, boolean recv) {
 		return endpoint;
 	}
 
+	private void reApplyIceCandiates(WebRtcEndpoint endpoint, boolean recv) {
```

Comment:
I would put it inline :)

Reply:
I think this method may grow. We may also do this re-applying of queued iceCandidates for the other webRTCEndpoints 
=> In the Kurento Example they also queue candidates on the "listener" side and re-apply those once the listener side has been established.